### PR TITLE
renderdoc: Fixes build with missing SWIG_PACKAGE

### PR DESCRIPTION
**Pull request by bkchr**
###### Motivation for this change
Required by nox-review for https://github.com/NixOS/nixpkgs/pull/32100

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/graphics/renderdoc/default.nix [hunk 1]
```diff
@@ -17,8 +17,14 @@ stdenv.mkDerivation rec {
   buildInputs = [
     qtbase qtsvg xorg.libpthreadstubs xorg.libXdmcp qtx11extras vulkan-loader
   ];
+
   nativeBuildInputs = [ cmake makeWrapper pkgconfig python3 bison autoconf automake pcre ];
 
+  SWIG_PACKAGE = fetchurl {
+    url = "https://github.com/baldurk/swig/archive/renderdoc-modified-1.zip";
+    sha256 = "0xxvkkaa5rp2xnnms6i1i4kbvq9fj3is0kiawwdix4hfys16m4mz";
```
Comment (Mic92) [inline pkgs/applications/graphics/renderdoc/default.nix]: Is this url stable?

Comment (bkchr) [reply pkgs/applications/graphics/renderdoc/default.nix]: Yeah: https://github.com/baldurk/swig/releases/tag/renderdoc-modified-1

Comment (Mic92) [reply pkgs/applications/graphics/renderdoc/default.nix]: These tarballs are generated by github. On regeneration they might change also the content does not change. Therefor it is required `fetchzip` or even better `fetchFromGitHub`, which is a wrapper around `fetchzip`.

Comment (bkchr) [reply pkgs/applications/graphics/renderdoc/default.nix]: Okay, I already tried `fetchFromGitHub`, but gave up after some tries. I will try again, I think I have an idea was is wrong.

Comment (joachifm) [reply pkgs/applications/graphics/renderdoc/default.nix]: @bkchr What errors did you encounter?

**Comment by bkchr** [comment]
@Mic92 @joachifm, sorry for taking that long to update the pull request. Now it works and uses fetchFromGitHub.

**Comment by bkchr** [comment]
Reminder for @Mic92

**Pull request merged**